lib/create-response-handler.ts

```diff
@@ -86,11 +88,66 @@ export interface ResponseScenarios {
   onError: (error: Error) => any,
 }
 
+/**
+ * IdentityResponseScenarios
+ *
+ * An object enumerating scenarios that can occur when handling a response from
+ * Verify.
+ *
+ * Relying Parties should handle each of these scenarios appropriately.
+ *
+ * NOTE: Should be used by all new services (not using MSA)
+ */
```

Comment:
Do we need to change the comment above (and its ResponseScenarios counterpart) to differentiate ResponseScenarios and IdentityResponseScenarios more clearly?

Reply:
"The `IdentityResponseScenarios` enumeration is relevant if your service connects to GOV.UK Verify using the Verify Service Provider (VSP) without a Matching Service Adapter (MSA)."